src/core/Akka.Streams/ActorMaterializer.cs

```diff
@@ -86,31 +121,21 @@ public static Config DefaultConfig()
         /// <returns>TBD</returns>
         public static ActorMaterializer Create(IActorRefFactory context, ActorMaterializerSettings settings = null, string namePrefix = null)
         {
-            var haveShutDown = new AtomicBoolean();
             var system = ActorSystemOf(context);
+            
+            // forces settings to get injected into the ActorSystem the first time we materialize
+            var defaultMaterializer = DefaultMaterializer.Get(system).Materializer;
```

Comment:
This will now return the cached, singleton materializer if we're running using the `ActorSystem` as our `context`.

If we're using a child actor as the context, then we still need to create a new materializer unfortunately